pkg/autosync/autosync.go

```diff
@@ -12,20 +12,19 @@ import (
 // in case where user specific values that are defined in granted.yml's `templateValues` are not available.
 // this is done so that users are aware of required keys when granted's credential-process is used through the AWS CLI.
 func Run(ctx context.Context, interactive bool) {
-	if registry.IsOutdatedConfig() {
-		clio.Warn("Outdated Profile Registry Configuration. Use `granted registry migrate` to update your configuration.")
-
-		clio.Warn("Skipping Profile Registry sync.")
-
-		return
-	}
 
 	registries, err := registry.GetProfileRegistries(interactive)
 	if err != nil {
 		clio.Debugf("unable to load granted config file with err %s", err.Error())
 		return
 	}
 
+	if len(registries) > 0 {
```

Comment:
`registry.IsOutdatedConfig()` checks a different field in the Granted config to the one in `registry.GetProfileRegistries()`.

IsOutdatedConfig checks if the length of `ProfileRegistryURLS` is > 0, whereas `GetProfileRegistries` uses the `ProfileRegistry` field.

The change you have made here will cause valid profile registry configuration to return the error. Could you please restore the previous behaviour? 

In this instance, I'd propose just restoring the `registry.IsOutdatedConfig` function and restoring the call to the function. The code there was working in production, so there is no need to change it IMO.